qiskit/terra/pulse_schedules.ipynb

```diff
@@ -0,0 +1,845 @@
+      "text/plain": [
+       "<Figure size 720x864 with 1 Axes>"
+      ]
+     },
+     "execution_count": 54,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "union_sched.draw()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 55,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAlkAAAKrCAYAAADCophVAAAABHNCSVQICAgIfAhkiAAAAAlwSFlzAAALEgAACxIB0t1+/AAAADh0RVh0U29mdHdhcmUAbWF0cGxvdGxpYiB2ZXJzaW9uMy4xLjAsIGh0dHA6Ly9tYXRwbG90bGliLm9yZy+17YcXAAAgAElEQVR4nO3db4xl533Q8d9vZvaf117bySZu4rokaZqC+KPU2UIRtBQoFfCCFtQCRYjyBvOCoiAQtOIFLf8kqChCIFHkQqUWASGlf3mDKKJtgiglbtOSltAS2rRp49rZOv6zXnvXO/fhxdyZbNazu3fnnuec5zzn85E8tsfjO2fOPnPP9zznuedmKSUAABjWztQbAADQI5EFAFCByAIAqEBkAQBUILIAACrYG/ObZeYTEfFERMT58+ff8653vWvMbw8AcCIf/vCHL5dS3nQv/09OdQuHxx9/vPzYBz44yfcGALgXFx64/ydLKZfu5f9xuRAAoAKRBQBQgcgCAKhAZAEAVCCyAAAqEFkAABWILACACkQWAEAFIgsAoAKRBQBQgcgCAKhAZAEAVCCyAAAqEFkAABWILACACkQWAEAFIgsAoAKRBQBQgcgCAKhAZAEAVCCyAAAqEFkAABWILACACkQWAEAFIgsAoAKRBQBQgcgCAKhAZAEAVCCyAAAqEFkAABWILACACkQWAEAFIgsAoAKRBQBQgcgCAKhAZAEAVCCyAAAqEFkAABWILACACkQWAEAFIgsAoAKRBQBQgcgCAKhAZAEAVCCyAAAqEFkAABWILACACkQWAEAFIgsAoAKRBQBQgcgCAKhAZAEAVCCyAAAqEFkAABWILACACkQWAEAFIgsAoAKRBQBQgcgCAKhAZAEAVCCyAAAqEFkAABWILACACkQWAEAFIgsAoAKRBQBQgcgCAKhAZAEAVCCyAAAqEFkAABWILACACkQWAEAFIgsAoAKRBQBQgcgCAKhAZAEAVCCyAAAqEFkAABWILACACkQWAEAFo0ZWZj6RmU9l5lOXL18e81sDAIxq1MgqpTxZSrlUSrl08eLFMb81AMCoXC4EAKhAZAEAVCCyAAAqEFkAABWILACACkQWAEAFIgsAoAKRBQBQgcgCAKhAZAEAVCCyAAAqEFkAABWILAC
```

Comment:
Should it be `"Schedules may be executed in a similar manner to circuits"`?

Reply:
yes 🤦‍♂